utils/acquireOpenSSL.js

```diff
@@ -42,10 +42,10 @@ class HashVerify extends stream.Transform {
 }
 
 // currently this only needs to be done on linux
-const applyOpenSSLPatches = async (buildCwd) => {
+const applyOpenSSLPatches = async (buildCwd, operatingSystem) => {
   try {
     for (const patchFilename of await fse.readdir(opensslPatchPath)) {
-      if (patchFilename.split(".").pop() === "patch") {
+      if (patchFilename.split(".").pop() === "patch" && patchFilename.split("-")[1] === operatingSystem) {
```

Comment:
What do you think about changing this so we can specify `all` for the OS on a patch? Looking at the new patch this is probably something we should include on all OSes anyway.

Reply:
I think this is a good idea. I'll switch the patch over to the new designation as well.